### ISSUE TITLE
[OUDS] Bullet list, multi-custom marker

### ISSUE THREAD

**Issue by MaxLardenois** [Orange-OpenSource/Orange-Boosted-Bootstrap#3054]
### Prerequisites

- [x] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

Make custom marker editable at the list item level 

### Motivation and context

Custom marker should be editable at the list item level as seen here: https://unified-design-system.orange.com/472794e18/p/49e7bf-bullet-list/t/027ad4abe2
